.gitignore

```diff
@@ -1,225 +1,225 @@
-
```

Comment:
There is bunch of dos line endings coming in somehow.  Do you know why? Can you fix them?  It would make this code review much shorter.